pkg/util/util_test.go

```diff
@@ -76,6 +76,21 @@ func TestGetImageURN(t *testing.T) {
 
 	out = GetImageURN("registry.io//", "test/this/image")
 	require.Equal(t, "registry.io/image", out)
+
+	UpdateCommonRegistries([]string{"k8s.gcr.io"})
+	out = GetImageURN("registry.io//", "k8s.gcr.io/pause:3.1")
+	require.Equal(t, "registry.io/pause:3.1", out)
+
+	// Update it again, now k8s.gcr.io should be deleted from common registries.
+	UpdateCommonRegistries([]string{"gcr.io"})
+	out = GetImageURN("registry.io", "k8s.gcr.io/pause:3.1")
+	require.Equal(t, "registry.io/k8s.gcr.io/pause:3.1", out)
+
+	out = GetImageURN("registry.io//", "k8s.gcr.io/pause:3.1")
+	require.Equal(t, "registry.io/pause:3.1", out)
```

Comment:
Should this have been `registry.io/k8s.gcr.io/pause` as common registries now only has `gcr.io`?

Reply:
With double slashes as suffix in customRegistry (registry.io//), it always replace the rep, it does not look at common registries, this was old behavior.

Reply:
Oh right, I forgot that part.